parent/pom.xml

```diff
@@ -59,7 +59,6 @@
         <commons.codec.version>1.11</commons.codec.version>
         <commons.collections.version>3.2.2</commons.collections.version>
         <commons.lang.version>2.6</commons.lang.version>
-        <cron4j.version>2.2.5</cron4j.version>
```

Comment:
You should also change ignite/pom.xml 
and move <module>modules/schedule</module> from lgpl profile
and add to <modules>

Reply:
done.